src/rgw/rgw_log.h

```diff
@@ -13,6 +13,66 @@
 
 #define dout_subsys ceph_subsys_rgw
 
+class RGWOp;
+
+struct delete_multi_obj_entry {
+  std::string key;
+  std::string version_id;
```

Comment:
put the string decls together, then http_status, then the two bool members (error and delete_marker), reorder the encode and decode to match

Reply:
thanks @mattbenjamin, updated!